poc/opaque_core.sage

```diff
@@ -27,9 +27,11 @@ class OPAQUECore(object):
     def __init__(self, config):
         self.config = config
 
-    def derive_random_pwd(self, pwdU, response, blind):
+    def derive_random_pwd(self, pwdU, response, blind, info):
         oprf_context = SetupBaseClient(self.config.oprf_suite)
-        y = oprf_context.finalize(pwdU, blind, response.data, None, None)
+        if info == None:
```

Comment:
Rather than do this check inline, can we move it to the calling code and assume that `info` is never `None`?

Reply:
info is None when the client identity is not given (e.g. like in the test vectors)

Reply:
oh, you mean assume it's never `None` just in this function ?